Docs 6247 visualizer get started ia revamp (#47)
mulesoft/docs-visualizer: modules/ROOT/pages/cloudhub-setup.adoc

* Mule Version 3.8.7 patch releases available after August 14, 2019
* Mule Version 3.9 patch releases available after April 5, 2019
* Mule Version 4.1 patch releases available after March 22, 2019
* Mule Version 4.2

To enable the Anypoint Monitoring agent:

. Verify that you are a user with Anypoint Monitoring User permission.
. Sign in to Anypoint Platform and click *Anypoint Monitoring*.
. Click *Settings*.
. Select *CloudHub*.
. From the *Environment* drop-down list, choose an environment.
. In *List of resources in <Environment Name>*, search for an app to visualize.
. Next to the app, click *Enable & Apply* to enable visualization and monitoring for the app.

For more information about Anypoint Monitoring, see xref:monitoring::enable-apps-deployed-to-cloud.adoc[Enable Monitoring Using the UI].

== Dedicated Load Balancers

Anypoint Visualizer supports dedicated load balancers for CloudHub apps using Mule version 3.9.x (version released April 5, 2019 or later).

[NOTE]
Anypoint Visualizer cannot correctly resolve dedicated load balancer connections for CloudHub apps using Mule 4 runtime versions at this time.

== Manage Performance and Data Collection

> The agent that Anypoint Visualizer uses to collect metadata may have a performance impact.

Reviewer: ```suggestion
The agent that Anypoint Visualizer uses to collect metadata sometimes has a performance impact.
```
(to avoid may-might-can)

> For more information on data collection and performance, see xref:visualizer::technical.adoc[Manage Performance and Data Collection].

Reviewer: ```suggestion
For more information about data collection and performance, see xref:visualizer::technical.adoc[Manage Performance and Data Collection].
```
GLOBALLY change "information on" to "information about"
